apps/src/sites/code.org/pages/views/share_privacy.js

```diff
@@ -25,5 +25,27 @@ $(document).ready(function() {
       study_group: 'share_student_privacy',
       event: 'printed'
     });
+
+    // Prevent repeatedly adding this iframe if button pressed repeatedly
+    if (!$('#iFramePdf').length) {
+      let pdfToPrintHash = {
+        '/privacy/student-privacy': '/files/privacy-policy-csf.pdf',
+        '/privacy/student-privacy-csd': '/files/privacy-policy-csd.pdf',
+        '/privacy/student-privacy-csp': '/files/privacy-policy-csp.pdf'
+      };
+
+      let pdfToPrint = pdfToPrintHash[window.location.pathname];
+
+      let iFramePdf = $(
+        '<iframe id="iFramePdf" src=' +
+          pdfToPrint +
+          ' style="display:none;"></iframe>'
+      );
+      $('body').append(iFramePdf);
+    }
+
+    let getPrintFrame = document.getElementById('iFramePdf');
+    getPrintFrame.focus();
+    getPrintFrame.contentWindow.print();
```

Comment:
To test on different browsers: Does the browser always wait for the PDF to be loaded before trying to print?

Reply:
Tried on Firefox and Safari and ran into a couple problems:
1. If you immediately try to print on page load (in either browser), it prints a blank page.
2. On Firefox, if you wait long enough for it to load, you get `SecurityError: Permission denied to access property "print" on cross-origin object`.

Reply:
Welp, this is a bummer:

https://github.com/mozilla/pdf.js/issues/5397